packages/compile-solidity/src/compilerSupplier/rangeUtils.ts

```diff
@@ -16,7 +17,8 @@ export function resolveToRange(version?: string): string {
   //if version was native or local, must determine what version that
   //actually corresponds to
   if (version === "native") {
-    return new Native().load().version();
+    const nativePath = TruffleConfig.default().compilers.solc.nativePath;
+    return new Native(nativePath).load().version();
```

Comment:
Hm, this always uses the default path.  Should this function take an optional second argument so it can be more accurate, and the uses of it updated as necessary/possible?